library/core/class.cache.php

```diff
@@ -23,6 +23,11 @@ abstract class Gdn_Cache {
     /** @var string Type of cache this this: one of CACHE_TYPE_MEMORY, CACHE_TYPE_FILE, CACHE_TYPE_NULL. */
     protected $cacheType;
 
+    /**
+     * @var null The current cache config.
+     */
+    private $activeConfig = null;
```

Comment:
This was a static variable of a method below. I extracted it to a property for easier testing.